install/mysql/plugin_formcreator_empty.sql

```diff
@@ -11,22 +11,6 @@ CREATE TABLE IF NOT EXISTS `glpi_plugin_formcreator_answers` (
   INDEX `plugin_formcreator_questions_id` (`plugin_formcreator_questions_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
 
-CREATE TABLE IF NOT EXISTS `glpi_plugin_formcreator_categories` (
-  `id`                               int(11) NOT NULL AUTO_INCREMENT,
-  `name`                             varchar(255) NOT NULL DEFAULT '',
-  `comment`                          text,
-  `completename`                     varchar(255) DEFAULT NULL,
-  `plugin_formcreator_categories_id` int(11) NOT NULL DEFAULT '0',
-  `level`                            int(11) NOT NULL DEFAULT '1',
-  `sons_cache`                       longtext,
-  `ancestors_cache`                  longtext,
-  `knowbaseitemcategories_id`        int(11) NOT NULL DEFAULT '0',
-  PRIMARY KEY (`id`),
-  INDEX `name` (`name`),
-  INDEX `knowbaseitemcategories_id` (`knowbaseitemcategories_id`),
-  INDEX `plugin_formcreator_categories_id` (`plugin_formcreator_categories_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
need to drop inc/category.class.php, front/category.*.php and remove the function plugin_formcreator_getDropdown() in hook.php